user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataDetailsComponent.java

```diff
@@ -1,46 +1,52 @@
 package life.qbic.datamanager.views.projects.project.rawdata;
 
-import static life.qbic.logging.service.LoggerFactory.logger;
-
-import com.vaadin.flow.component.grid.dataview.GridLazyDataView;
-import com.vaadin.flow.component.tabs.TabSheet;
-import com.vaadin.flow.data.provider.AbstractDataView;
-import com.vaadin.flow.data.provider.SortDirection;
+import com.vaadin.flow.data.provider.DataProvider;
+import com.vaadin.flow.data.provider.QuerySortOrder;
 import com.vaadin.flow.data.renderer.ComponentRenderer;
-import com.vaadin.flow.spring.annotation.SpringComponent;
-import com.vaadin.flow.spring.annotation.UIScope;
-import jakarta.annotation.security.PermitAll;
+import java.io.BufferedInputStream;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
 import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Date;
+import java.util.EnumMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
-import life.qbic.application.commons.SortOrder;
+import life.qbic.application.commons.FileNameFormatter;
 import life.qbic.datamanager.ClientDetailsProvider;
+import life.qbic.datamanager.files.export.download.DownloadStreamProvider;
+import life.qbic.datamanager.files.export.rawdata.RawDataUrlFile;
+import life.qbic.datamanager.files.export.rawdata.RawDataUrlFile.RawDataURL;
 import life.qbic.datamanager.views.Context;
 import life.qbic.datamanager.views.GridDetailsItem;
+import life.qbic.datamanager.views.UiHandle;
```

Comment:
Hi Sven
As we discovered that the ui problem was not a problem with the UI access or accesslater methods but a misconfiguration on our part, I would like to talk to you about the UiHandle again. As far as I can tell we can remove it as it is no longer needed. This discussion is not part of this PR, and we can tackle it separately.